scripts/exregional_run_prepstart.sh

```diff
@@ -365,6 +365,46 @@ else
   fi
 
 fi 
+
+#
+#-----------------------------------------------------------------------
+#
+# condut surface surgery to transfer RAP/HRRR surface fields into RRFS.
+#
```

Comment:
suggest to add more comments:
 # This is usually done once or twice per year by experienced users such as Ming, Tanya. 
 #  Consult Ming or Tanya first before turning on this surgery.